src/main/services.js

```diff
@@ -79,10 +79,11 @@ exports.ServiceManager = class {
         if (!isRunning) {
           return Promise.try(init)
             .then(() => {
-              const child = execFile(options.executable, options.arguments || [], {
+              const child = spawn(options.executable, options.arguments || [], {
```

Comment:
Seems like an excellent idea. A much better choice
https://www.freecodecamp.org/news/node-js-child-processes-everything-you-need-to-know-e69498fe970a/